app/models/concerns/hyrax/admin_set_behavior.rb

```diff
@@ -83,6 +83,13 @@ def active_workflow
       Sipity::Workflow.find_active_workflow_for(admin_set_id: id)
     end
 
+    # Calculate and update who should have edit access based on who
+    # has "manage" access in the PermissionTemplateAccess
+    def update_access_controls!
+      update!(edit_users: permission_template.access_grants.where(access: 'manage', agent_type: 'user').pluck(:agent_id),
```

Comment:
Can you extract local variables for these two lines. They are rather long and hard to quickly parse.

```ruby
new_edit_users = permission_template.access_grants.where(access: 'manage', agent_type: 'user').pluck(:agent_id)
new_edit_groups = permission_template.access_grants.where(access: 'manage', agent_type: 'group').pluck(:agent_id)
update!(edit_users: new_edit_users, edit_groups: new_edit_groups)
```